### PR TITLE
docs: Fix documentation according to user feedback

### PR DESCRIPTION
**Pull request by Paula-Kli**
In the jupyter section was a video that I was not able to open locally neither did it play in the docs - is that just me or a general problem?

Also please read the update of the capella model carefully - not sure whether everything is correct in there

### CODE REVIEW

File: docs/docs/user/projects/models/create.md [hunk 1]
```diff
@@ -101,6 +101,12 @@ name for the repository and project.
...
-restricted. You will not be able to setup backups or create read-only sessions.
+This is an important step.
+
+Here, you can select the version and the model nature of your tool. If you
+don't select any version, the functionality will be restricted. You will not be
+able to setup backups or create read-only sessions.
+
+For Capella the best version to pick currently is 6.0.0. Older versions are
+outdated and the possibility might to use them might be removed in the future.
+Make sure your model is up to date and you update it in time.
```
Comment (MoritzWeber0) [inline docs/docs/user/projects/models/create.md:112]: We should not give any recommendations about specific versions. They become outdated pretty soon.

File: docs/docs/user/projects/models/update.md [hunk 2]
```diff
@@ -3,22 +3,28 @@
  ~ SPDX-License-Identifier: Apache-2.0
  -->
 
-# Update a Model to a Newer Version
+# Update a Capella Model to a Newer Version
```
Comment (MoritzWeber0) [inline docs/docs/user/projects/models/update.md:6]: As it's tool specific now it belongs to the tool specific docs (tools > capella)

File: docs/docs/user/projects/models/update.md [hunk 3]
```diff
@@ -3,22 +3,28 @@
...
+   [Change model metadata](metadata.md)
 1. Create a new session with the new Capella version.
-1. Migrate the model in your local workspace.
+1. In your persistent workspace loaded, execute the following steps:
+   <!-- prettier-ignore -->
+    1. Right click on the model in your workspace.
+    1. Select `Migration` and `Migrate Project toward current version`
+       ![Migrate Capella model](../../tools/capella/teamforcapella/migrate-capella-model.png)
+    1. A backup is  required.
+       ![Disable backup option](../../tools/capella/teamforcapella/backup-migration.png)
```
Comment (MoritzWeber0) [inline docs/docs/user/projects/models/update.md:30]: In the info-box, you wrote: "If you want to update a Capella model saved in TeamForCapella, please follow the Capella specific instructions instead".

This is why I assume that this information is just about local Capella models in the workspace. But in these steps, you link to the TeamForCapella guide.

**Review by MoritzWeber0 — CHANGES_REQUESTED** [review]

File: docs/docs/user/projects/models/update.md (hunk 3 above)
Comment (Paula-Kli) [reply docs/docs/user/projects/models/update.md:30]: so shall I remove the link to the backups?

File: docs/docs/user/tools/capella/update.md
Comment (dominik003) [inline docs/docs/user/tools/capella/update.md:1]: For me this belongs inside the `Tools/capella` section as it is tool specific. We can then just add links to the tool specific update instructions here.

**Review by dominik003 — CHANGES_REQUESTED** [review]
Thanks for updating the docs. I have just a few comments and after they are resolved we can merge this one.

File: docs/docs/user/projects/models/update.md (hunk 3 above)
Comment (Paula-Kli) [reply docs/docs/user/projects/models/update.md]: it is moved to `tools > capella` as it was about updating models and is now capella specific
so it is the file on which you commented later on

File: docs/docs/user/projects/models/metadata.md [hunk 4]
```diff
@@ -5,6 +5,10 @@
 
 # Update Model Metadata (Version and Nature)
 
+<!-- prettier-ignore -->
+!!! warning
+    Make sure to read the documentation about updating a [TeamForCapella model](../../tools/capella/teamforcapella/update.md) or about updating a [Capella model](./update.md) first!
```
[the remark this answers is not in the document]
Comment (Paula-Kli) [reply docs/docs/user/projects/models/metadata.md:10]: good catch! thanks!

File: docs/docs/user/tools/capella/update.md
Comment (Paula-Kli) [reply docs/docs/user/tools/capella/update.md:1]: what do you mean by this? it is in `tools/capella`?

Comment (dominik003) [reply docs/docs/user/tools/capella/update.md:1]: True, you are right. I guess I just misread the path

File: docs/docs/user/projects/models/create.md [hunk 5]
```diff
@@ -101,6 +101,12 @@ name for the repository and project.
...
-nature of your tool. If you don't select any version, the functionality will be
-restricted. You will not be able to setup backups or create read-only sessions.
+This is an important step.
+
+Here, you can select the version and the model nature of your tool. If you
+don't select any version, the functionality will be restricted. You will not be
+able to setup backups or create read-only sessions.
+
+If suitable for your project, select one of the latest versions of Capella.
+Older versions are outdated and may be removed in the future. Make sure your
```
Comment (amolenaar) [inline docs/docs/user/projects/models/create.md:111]:
```suggestion
Older versions may be removed in the future. Make sure your
```

**Review by amolenaar — APPROVED** [review]
It looks more clear this way.

One small improvement suggestion.

File: docs/docs/user/projects/models/create.md (hunk 5 above)
Comment (Paula-Kli) [reply docs/docs/user/projects/models/create.md]: thanks!

File: docs/docs/user/tools/capella/update.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+1. Terminate the session.
+1. Update the model version to the newer Capella version:
+   [Change model metadata](metadata.md)
+1. Create a new session with the new Capella version.
+1. In your persistent workspace loaded, execute the following steps:
+   <!-- prettier-ignore -->
+    1. Right click on the model in your workspace.
+    1. Select `Migration` and `Migrate Project toward current version`
+       ![Migrate Capella model](../../tools/capella/teamforcapella/migrate-capella-model.png)
+    1. A backup is  required.
```
Comment (MoritzWeber0) [inline docs/docs/user/tools/capella/update.md:29]:
```suggestion
    1. A backup is recommended.
```

It's not required or technically enforced, but definitely recommended.

**Review by MoritzWeber0 — APPROVED** [review]
Thanks for the improvement, looks good in general :) 

There is a new Github action to check for mkdocs warnings, please rebase on main to get it in this PR.

In addition, I have two small comments. Once they are resolved, the PR can be merged.

File: docs/mkdocs.yml
[the remark this answers is not in the document]
Comment (Paula-Kli) [reply docs/mkdocs.yml:1]: should all be fixed - the alerts were in the user docs but as only admins can create them and there was still a link to what seems the same alerts folder structure I moved them to the admin section again

[the remark this answers is not in the document]
Comment (Paula-Kli) [reply docs/mkdocs.yml:1]: just as a heads up why that happened

**Pull request merged**